projects/sbb-esta/angular-public/src/lib/checkbox/_checkbox.scss

```diff
@@ -65,6 +65,7 @@ $checkBoxContainerBorder: 2;
           left: 4px;
           top: 8.5px;
           border: 1px solid $sbbColorGranite;
+          background-color: $sbbColorGranite;
```

Comment:
Why was this necessary?

Reply:
This might be related to #141 . On my screen the checkbox was bigger then it should be and the indeterminate state was displayed only as a grey border instead of a filled grey box. This might be  redundant, when #141 is fixed and the checkbox always has the pre-defined size.